### ISSUE TITLE
link in table does not work

### ISSUE THREAD

**Issue by Athenacle** [xiaoxinghu/orgajs#1]
with the lastest version  0.2.7 on npm, links in a table cell does not work

```javascript
const { parse } = require(`orga`);

const orgLink = `[[https://some.link][Some Link]]`;

var ast = parse(orgLink);

console.log(ast.children[0].children[0]); // this works fine

const orgLinkInTable = `
| Text            | [[http://I.should.be.link][link in header]]                            |
|-------------+---------------------------------------------------------------|
| Text   Text  | [[https://I.should.be.link][link in row]]                                |
`;

ast = parse(orgLinkInTable);

console.log(ast.children[0].children[0].children[1].children[0]); // link in header
console.log(ast.children[0].children[2].children[1].children[0]); // link in body

```
output:
``` javascript
{ type: 'link',
  children: [],
  uri:
   { raw: 'https://some.link',
     protocol: 'https',
     location: '//some.link' },
  desc: 'Some Link',
  parent:
   { type: 'paragraph',
     children: [ [Circular] ],
     parent: { type: 'root', children: [Array], meta: {} } } }


{ type: 'text',
  children: [],
  value: '[[http://I.should.be.link][link in header]]',
  parent:
   { type: 'tableCell',
     children: [ [Circular] ],
     parent: { type: 'tableRow', children: [Array], parent: [Object] } } }


{ type: 'text',
  children: [],
  value: '[[https://I.should.be.link][link in row]]',
  parent:
   { type: 'tableCell',
     children: [ [Circular] ],
     parent: { type: 'tableRow', children: [Array], parent: [Object] } } }

```

**Comment by xiaoxinghu** [comment]
@Athenacle Thanks for the feedback. Fixed it in a79de57144abc51cced56d19990d6ee64a2f3839. Checkout the latest version (0.3.1). Let me know if it doesn't work.

**Comment by Athenacle** [comment]
I install version 0.3.1, it works, thank you
